fusion-react/package.json

```diff
@@ -40,7 +40,8 @@
   },
   "dependencies": {
     "@rtsao/react-ssr-prepass": "^1.0.5",
-    "prop-types": "^15.7.2"
+    "prop-types": "^15.7.2",
+    "react-is": "^16.8.0"
```

Comment:
`react-is` is a peerDependency of `react-ssr-prepass`. By adding it here, it is no longer required for the consumer to install it